lib/core/src/Cardano/Wallet/Primitive/CoinSelection/Balance.hs

```diff
@@ -510,15 +515,6 @@ performSelection minCoinFor costFor bundleSizeAssessor criteria
                 , balanceRequired
                 }
   where
-    SelectionCriteria
-        { outputsToCover
-        , utxoAvailable
-        , selectionLimit
-        , extraCoinSource
-        , assetsToMint
-        , assetsToBurn
-        } = criteria
```

Comment:
Hi @rvl.

I'd like to ask a favour: would it be possible to avoid changing these to use `RecordWildCards`, at least for the moment? 

There are two things here:
- it makes it harder to figure out where fields have come from, as some field names are shared by multiple records. If we use record wildcards, then go-to-definition no longer works.
- I'm currently working on this module, so it would help me greatly if we could keep any changes to a minimum!

Thanks!